### DISCUSSION TITLE
Athome WLED  High Power Addressable LED Strip Controller

### DISCUSSION THREAD

**Discussion by neandr** [arendst/Tasmota#16338] [Q&A] [answered]
Just found this device:
> Athome WLED  High Power Addressable LED Strip Controller
https://www.athom.tech/blank-1/wled-high-power-led-strip-controller

Anyone knows about it .. and does it support Tasmota .. as other devices offered by Athome does?

**Comment by sfromis** [comment, marked as answer]
As answered on Discord, yes it is known and supported....
https://templates.blakadder.com/athom_LS4P-TAS.html
